packages/go_router_builder/example/lib/main.g.dart

```diff
@@ -52,7 +58,11 @@ extension $HomeRouteExtension on HomeRoute {
 }
 
 extension $FamilyRouteExtension on FamilyRoute {
-  static FamilyRoute _fromState(GoRouterState state) => FamilyRoute(
+  static FamilyRoute _fromState(GoRouterState state) => const FamilyRoute();
```

Comment:
This feel a bit unnecessary? maybe we don't need to generate extension for shellRoute at all?

Reply:
This change got reverted with the changes on example/lib/main.dart but if you're talking more generally about the `ShellRouteExtension._fromState`, it is passed as the factory of `ShellRouteData.$route` to create `$appRoutes`.